packages/blade-coverage-extension/background_script.js

```diff
@@ -40,17 +40,27 @@ const calculateBladeCoverage = () => {
     // If element has data-blade-component add it
     if (elm.hasAttribute('data-blade-component')) {
       bladeNodeElements.push(elm);
-      // If element has a parent node which has `data-blade-component` attribute
-      // Then count this node too.
-    } else if (elm.closest('[data-blade-component]')) {
-      bladeNodeElements.push(elm);
     }
   });
 
   const totalNodes = totalNodeElements.length;
   const bladeNodes = bladeNodeElements.length;
   const bladeCoverage = Number(((bladeNodes / totalNodes) * 100).toFixed(2));
 
+  if (shouldHighlightNodes) {
+    totalNodeElements.forEach((node) => {
+      node.style.outline = '1px solid rgba(255, 0, 0, 0.5)';
+    });
+
+    bladeNodeElements.forEach((node) => {
+      node.style.outline = '1px solid green';
```

Comment:
can we just highlight non-blade nodes? otherwise it'll lead to a lot of clutter for consumers to figure out what they shall focus upon

Reply:
makes sense.